fuzzers/LIFCL/070-iologic_mode/fuzzer.py

```diff
@@ -46,30 +48,82 @@
 
     ("IOL_R3B", "SIOLOGICB", FuzzConfig(job="IOL3DEMODE", device="LIFCL-40", sv="../shared/empty_40.v", tiles=["CIB_R3C87:SYSIO_B1_DED"])),
 
+    # LIFCL-17 tiles
+    ("IOL_T57A", "SIOLOGICA", FuzzConfig(job="IOLT57AMODE", device="LIFCL-17", sv="../shared/empty_17.v", tiles=["CIB_R0C57:SYSIO_B0_0_15K"])),
```

Comment:
the tiletypes here don't look correct; did you actually have a go at running the fuzzer?

Reply:
I'm pretty sure that they are or. I took them from the docs: https://gatecat.github.io/prjoxide-html/LIFCL/LIFCL-17/index.html.

I was able to get solution for `SYSIO_B0_0_15K` and `SYSIO_B1_0_15K`

Reply:
oh, I see, you've added multiple tiles of the same type? that shouldn't be necessary, I think

Reply:
Right, I wasn't sure what's the minimum number of tiles required. What do you suggest? I added all of them to catch possible discrepancies during fuzzing.

Reply:
For bitstream config, only one set of tiles is always safe, and to minimise fuzzer runtime is the way to go (occasionally this is different for routing but that doesn't apply here).

Once this is sorted I'll give it a run, push the new DB and merge.

Reply:
Ok. I left only one tile of each type. I'm getting solution for "regular" `SYSIO_*_15K` but unfortunately still not for `SYSIO_B1_DED_15K`.

Reply:
Thanks, I'll have a look at how it runs now.

Reply:
Looking at unknown bits in a design that uses an ODDR on that pin, it seems like like these bits end up in `CIB_R4C75:PIC_B1_DED_15K`, I've updated the fuzzer accordingly and will see how it goes.